### PR TITLE
Landing Page (Merge)

### PR DESCRIPTION
**Pull request by LRRoberts0122**
Most of the landing page (see issue #3) is done, minus a few possible tweaks here and there. =)

Some of the new components with this PR include:

- GhostButton
image
image

-  Landing Page (see below)

The individual Landing Page components consist of:

- Header Component
image

- The Logistics Wizard component (Logistics Wizard section)
image

- Icon Section component
image

- An Architecture Diagram component
image

- Footer Component
image

*Known Issues*
- Need to fix styling issues on Ghost Buttons with Links. (Issue #39)
- Need to add Github Icon (Issue #24)

### CODE REVIEW

File: src/components/GhostButton/GhostButton.story.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { storiesOf, action } from '@kadira/storybook';
+import GhostButton from './GhostButton';
+
+storiesOf('GhostButton', module)
+  .add('default', () => (
+    <GhostButton
+      clicky={action('You clicked the button.')}
+    />
+  ))
+  .add('custom prop', () => (
```
Comment (colbycheeze) [inline src/components/GhostButton/GhostButton.story.js]: Can you remove this story.

File: src/routes/Home/components/LandingPage/ArchDiagram/ArchDiagram.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.archDiagram {
+  position: relative;
+  overflow: hidden;
+  padding: 7rem 12rem 10rem 12rem;
+
+  background: rgb(15,148,167);
+  background: -moz-linear-gradient(top,  rgb(15,148,167) 0%, rgb(5,116,132) 100%);
```
Comment (colbycheeze) [inline src/routes/Home/components/LandingPage/ArchDiagram/ArchDiagram.scss]: We dont' need these, have autoprefixer in the build

File: src/routes/Home/components/LandingPage/Footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  }
+}
+
+.gradient {
+  height: 2px;
+  width: 100%;
+  margin-bottom: 2rem;
+
+  background: rgb(145, 195, 131);
+  background: -moz-linear-gradient(left,  rgb(255,255,255) 0%, rgb(145,195,131) 100%);
```
Comment (colbycheeze) [inline src/routes/Home/components/LandingPage/Footer/Footer.scss]: This one too.

File: src/styles/base/_colors.min.css [hunk 4]
```diff
@@ -0,0 +1 @@
+:export{primary1Color:#0F94a7}:export{primary2Color:#485566}:export{accent1Color:#91C383}:export{alternateTextColor:#fff}:export{borderColor:#DADDE0}:export{textColor:#485566}
```
Comment (colbycheeze) [inline src/styles/base/_colors.min.css]: Can you delete any minified files that your editor has generated.

File: src/components/GhostButton/GhostButton.story.js (hunk 1 above)
Comment (LRRoberts0122) [inline src/components/GhostButton/GhostButton.story.js]: Fixed.

File: src/routes/Home/components/LandingPage/ArchDiagram/ArchDiagram.scss (hunk 2 above)
Comment (LRRoberts0122) [inline src/routes/Home/components/LandingPage/ArchDiagram/ArchDiagram.scss]: Fixed!

File: src/routes/Home/components/LandingPage/Footer/Footer.scss (hunk 3 above)
Comment (LRRoberts0122) [inline src/routes/Home/components/LandingPage/Footer/Footer.scss]: Fixed

File: src/routes/Home/components/LandingPage/Header/Header.scss [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+.title {
+  color: rgb(255, 255, 255);
+  font-weight: 600;
+}
+
+.topLink {
+  position: relative;
+  z-index: 20;
+
+  & .text {
```
Comment (LRRoberts0122) [inline src/routes/Home/components/LandingPage/Header/Header.scss]: Fixed

File: src/styles/base/_colors.min.css (hunk 4 above)
Comment (LRRoberts0122) [inline src/styles/base/_colors.min.css]: Fixed - also fixed the problem where the editor was generating compiled sass. =) Shouldn't be an issue anymore.

**Comment by colbycheeze** [comment]
Okay good stuff.

**Pull request merged**